theme/mixins/backdrop.less

```diff
@@ -1,11 +1,15 @@
-.backdrop() {
+.transparentBackdrop() {
   position: fixed;
   top: -999px;
   left: -999px;
   right: -999px;
   bottom: -9999px;
-  background-color: @sk-black;
-  opacity: 0.7;
   transition: opacity 0.3s;
   z-index: @z-index-inline-overlay;
 }
+
+.shadedBackdrop() {
```

Comment:
Should we rename this to `backdrop`?  Having a background colour seems like a sensible default, and this was the old API too.